package com.megajuegos.independencia.exceptions;

public class BattleNotFoundException extends RuntimeException{
}
